config/core/test-configs.yaml

```diff
@@ -2447,6 +2463,7 @@ test_configs:
     test_plans: &r8a7795-salvator-x_test-plans
       - baseline
       - baseline-nfs
+      - kselftest-alsa
```

Comment:
@montjoie This device type is only available in BayLibre's lab, please confirm it's OK to enable these tests on it.

Reply:
@montjoie @broonie No news and it's failing to boot now:
https://staging.kernelci.org/test/plan/id/62594298fa241475d805f521/

So I suggest we drop it for now.